src/components/characters/CharacterRows.tsx

```diff
@@ -12,6 +14,7 @@ function CharacterDiamond({
   active: boolean;
   labelTop: boolean;
 }): JSX.Element {
+  const router = useRouter();
```

Comment:
I don't think we are still using these ? (import use router and slugify)

Reply:
Oops